package.json

```diff
@@ -71,10 +71,10 @@
 		"layer-check": "flub check layers --info layerInfo.json",
 		"lint": "fluid-build --task lint",
 		"lint:fix": "fluid-build --task lint:fix",
-		"policy-check": "flub check policy",
+		"policy-check": "POLICY_CHECK=1 flub check policy",
```

Comment:
New env variable so jest configs don't try to import from test-tools, so we can run policy checks on them without having to build the repo.

Reply:
I also suggest using a fluid prefix in the env variable, eg FLUID_POLICY_CHECK.

Reply:
Done.